### PR TITLE
Add error page

### PR DESCRIPTION
**Pull request by isSerge**
image

- add error page
- specific errors are reflected in the error title
- body text is generic (until we clarify resolutions for specific errors)

### CODE REVIEW

**Review by nazar-pc — COMMENTED** [review]
I have a few questions about UI:
1. Do I understand correctly that button to show logs is unreachable when error happens and makes it difficult for user to assist us with figuring out what went wrong?
2. Interface can benefit from some hint as to how to contact the team and which way is preferred

**Comment by isSerge** [comment]
@nazar-pc 
1. You are right. Maybe we could have an error component as a whole page instead of modal. What do you think @achiurizo ?
2. We had a discussion on this and it will be added once all details are clarified

**Comment by achiurizo** [comment]
> 1. You are right. Maybe we could have an error component as a whole page instead of modal. What do you think @achiurizo ?

Yep that sounds fine with me :+1:

File: src/pages/Index.vue
Comment (ozgunozerk) [inline src/pages/Index.vue:74]: nice catch

**Review by ozgunozerk — DISMISSED** [review]
looking good to me, but let's wait for Nazar's approval

File: src/lib/client.ts
Comment (ozgunozerk) [inline src/lib/client.ts:165]: for `startFarming` and `startNode` -> you changed the signature from `Promise<boolean>` to `Promise<void>`.
We discussed it before, and if it will work like that, I'm pro for this simplification. 
However, the line `return tauri.invoke("farming", { path, rewardAddress, plotSize });` is still returning `true`/`false` (same for node), because the backend functions' signatures are the same.

Comment (isSerge) [reply src/lib/client.ts:165]: `invoke` returns `Promise<unknown>`, but we don't really care about value here, only if it resolves or not

Comment (isSerge) [reply src/lib/client.ts:165]: we don't use value in the `store.ts`: `await client.startFarming(this.plotDir, this.plotSizeGB);`

**Review by nazar-pc — APPROVED** [review]

**Pull request merged**
